src/components/Navigator/BaseNavigatorCardItem.vue

```diff
@@ -0,0 +1,116 @@
+<!--
+  This source file is part of the Swift.org open source project
+
+  Copyright (c) 2022 Apple Inc. and the Swift project authors
+  Licensed under Apache License v2.0 with Runtime Library Exception
+
+  See https://swift.org/LICENSE.txt for license information
+  See https://swift.org/CONTRIBUTORS.txt for Swift project authors
+-->
+
+<template>
+  <div
+    v-bind="$attrs"
```

Comment:
I think if you are binding to the top it should just pass them through, no need for `v-bind="$attrs"`.